core/Dockerfile.template

```diff
@@ -10,10 +10,11 @@ RUN npm ci
 FROM balenalib/%%BALENA_MACHINE_NAME%%-node:10-buster-build
 
 ENV UDEV=1
+ENV DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
```

Comment:
hey @rcooke-warwick I recently read this https://github.com/balena-io/docs/issues/591, any side-effects you know that can or did happen with having the DBUS path as env var

Reply:
@vipulgupta2048 I haven't seen any side effects yet! and have been running commands from the container